CMakeLists.txt

```diff
@@ -100,7 +100,7 @@ elseif(ANDROID)
     message(FATAL_ERROR "Android build not supported!")
 elseif(APPLE)
     target_compile_definitions(platform_wsi INTERFACE VK_USE_PLATFORM_MACOS_MVK VK_USE_PLATFORM_METAL_EXT)
-elseif(UNIX AND NOT APPLE) # i.e.: Linux
```

Comment:
My main issue is UNIX AND NOT APPLE also applies to Android.

You know it's hard to read since it has a comment specifying `# i.e : Linux`

Reply:
Only concern is that there may be another platform other than Linux or BSD that would fall under this case.

Reply:
Fuchsia, QNX, and DragonFly are the ones that are most likely to be hurt. Fuchsia & QNX has its own build, and DragonFly is a BSD variant, but yeah just wanted to make it known.

Reply:
- DragonFly BSD only has `DragonFly` but not `BSD` in `uname -s` output
- GNU/Hurd (9d38cccf33e0) has `GNU` but not `Linux` in `uname -s` output

Reply:
I don't understand Android concern when it was explicitly marked as unsupported in 2896ff0b6.

Reply:
@jbeich please verify the fix when you can.